src/parsing/test_binast.cc

```diff
@@ -20,6 +20,7 @@ static const char* test_scripts[] = {
   "'use strict';\nvar obj = {foo: 'bar', bar: 1, baz: function(x) { return x; }};",
   "'use strict';\nvar obj = {}; obj.foo = 'bar';",
   "'use strict';\nvar arr = [];",
+  "'use strict';\nconst square = (x) => x * x;",
```

Comment:
added sloppy mode and arrow function bc we seem to be able to parse them now

Reply:
Nice!